okta/resource_okta_app_group_assignments.go

```diff
@@ -0,0 +1,305 @@
+package okta
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/okta/okta-sdk-golang/v2/okta"
+	"github.com/okta/okta-sdk-golang/v2/okta/query"
+	"github.com/peterhellberg/link"
+	"github.com/pkg/errors"
```

Comment:
Instead of using `"github.com/pkg/errors"` just use `fmt` package (`fmt.Errorf("failed to do the thing: %v", err)`)